Agents/InequalityIndexCalculationAgent/python_scripts/README.md

```diff
@@ -0,0 +1,92 @@
+# Description
+
+The code and data collected in this folder is dedicated to reproduce the figures in **[Preprint 323](https://como.ceb.cam.ac.uk/preprints/323/): Impact of heat pumps and future energy prices on regional inequalities.**
+
+## Data
+Please download the data (should be two folder, called `Data` and `GB_shapefile` respectively) from the project folder in the Dropbox (`CoMo shared/jx309/c4e-jx309-heatpumps/Codes_and_Data`, or contact Jethro), and replace it with the `./Data` and `./GB_shapefile`.
```

Comment:
Replace "(should be two folder, ..." with "(there should be two folders called..."

Reply:
Replace "from the project folder in the Dropbox (`CoMo shared/jx309/c4e-jx309-heatpumps/Codes_and_Data`, or contact Jethro), and replace it with the `./Data` and `./GB_shapefile`." with "the archive for preprint 323 (available on request from the corresponding author) and use it to replace the contents of the `./Data` and `./GB_shapefile fiolders."